notifications_utils/version.py

```diff
@@ -1 +1 @@
-__version__ = '10.0.0'
+__version__ = '10.0.1'
```

Comment:
thinking of them as new/additional graphs. We can ship this and not affect graphs. The existing ones can be maintained by starting out the service id?